libshaderc_util/src/io_test.cc

```diff
@@ -53,6 +53,18 @@ TEST(IsAbsolutePathTest, Linux) {
   EXPECT_TRUE(IsAbsolutePath("/☢/g o/ogle"));
 }
 
+TEST(IsAbsolutePathTest, Windows) {
+  EXPECT_TRUE(IsAbsolutePath("\\\\Server1000\\superuser\\file"));
```

Comment:
Consider using [raw-string literals](http://en.cppreference.com/w/cpp/language/string_literal) here, for easier readability.